### PR TITLE
[MERGE] calendar: add link to join video call

### PR DESCRIPTION
**Pull request by kbapt**
This commit adds a link to each calendar.event to join a video call
hosted on Jitsi.

TaskID: 2367543

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/calendar/models/calendar_event.py [hunk 1]
```diff
@@ -78,6 +80,14 @@ def _default_partners(self):
                 partners |= self.env['res.partner'].browse(active_id)
         return partners
 
+    @api.model
+    def _default_videocall_location(self):
+        if self.env.context.get('calendar_no_videocall'):
+            return False
+        jitsi_url = self.env['ir.config_parameter'].sudo().get_param('calendar.jitsi_base_domain', 'https://meet.jit.si')
```
Comment (tde-banana-odoo) [inline addons/calendar/models/calendar_event.py:87]: Would say website_jitsi.jitsi_server_domain

File: addons/calendar/models/calendar_event.py [hunk 2]
```diff
@@ -78,6 +80,14 @@ def _default_partners(self):
...
+        jitsi_prefix = self.env['ir.config_parameter'].sudo().get_param('calendar.jitsi_prefix', 'odoo')
```
Comment (tde-banana-odoo) [inline addons/calendar/models/calendar_event.py:88]: Would avoid bloating config parameters

**Review by tde-banana-odoo — COMMENTED** [review]
According to our chair discussion

File: addons/calendar/models/calendar_event.py [hunk 3]
```diff
@@ -211,6 +222,7 @@ def _compute_is_highlighted(self):
          ('confidential', 'Only internal users')],
         'Privacy', default='public', required=True)
     location = fields.Char('Location', tracking=True, help="Location of Event")
+    videocall_location = fields.Char('Join Video Call', default=_default_videocall_location, readonly=True)
```
Comment (tivisse) [inline addons/calendar/models/calendar_event.py:225]: We could remove the readonly True, in case people already have a jitsi room for a meeating.

**Comment by tivisse** [comment]
@robodoo r+

**Comment by tivisse** [comment]
@robodoo merge

**Pull request merged**